test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -420,15 +428,20 @@ public static JMXServiceURL getJMXServiceURL() throws MalformedURLException
      */
     @BeforeClass
     public static void setUpClass()
+    {
+        prePrepareServer();
```

Comment:
this was pulled out of [CASSANDRA-19830](https://issues.apache.org/jira/browse/CASSANDRA-19830) to avoid conflict once that merges.  This style makes this patch easier, so wanted to match